src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -35,6 +35,14 @@ A recommended approach.
 * [You might not need Redux: The Flutter edition][], by Ryan Edge
 * [Making sense of all those Flutter Providers][]
 
+## Riverpod
+
+A provider rewrite by the same author, compiled safe, more testable, more flexible
+
+* [Riverpod Homepage][]
```

Comment:
The "h" in "homepage" should be lowercase, imho.

Also, consider moving the "homepage" outside the link, like this:

```suggestion
* [Riverpod][] homepage
```

I think it's cleaner. Of course, you'll need to update the named link below if you do this.